test/integration_test/test_subprocess.py

```diff
@@ -0,0 +1,116 @@
+import tempfile
+import os
+import subprocess
+import shutil
```

Comment:
この 2 行がいらないっぽいのですがどうなんでしょう…？
```suggestion
import os
import subprocess
```

Reply:
あーこれ、実験用のファイルですね...
playgroundに移動しておきます...！